tests/pytests/unit/test_version.py

```diff
@@ -54,6 +54,10 @@
             (3000, 2, "nb", 20201214010203, 0, "1e7bc8f"),
             "3000.2nb20201214010203",
         ),
+        ("v3006.0", (3006, 0, "", 0, 0, None), "3006.0"),
```

Comment:
```suggestion
        ("v3006.0", (3006, 0, "", 0, 0, None), "3006"),
        ("v3006.0", (3006, 0, "", 0, 0, None), "3006.0"),
```
Maybe also try to see if it handles bad tags?